var/spack/repos/builtin/packages/timemory/package.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Timemory(CMakePackage):
+    """Timing + Memory + Hardware Counter Utilities for C/C++/CUDA/Python"""
+
+    homepage = 'https://timemory.readthedocs.io/en/latest/'
+    git = 'https://github.com/NERSC/timemory.git'
+
+    version('master', branch='master', submodules=True)
+
+    variant('python', default=True, description='Enable Python support')
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('papi', default=True, description='Enable PAPI support')
+    variant('cuda', default=True, description='Enable CUDA support')
+    variant('cupti', default=True, description='Enable CUPTI support')
+    variant('caliper', default=True, description='Enable Caliper support')
+    variant('gperftools', default=True, description='Enable gperftools support')
+
+    extends('python', when='+python')
+    depends_on('python@3:', when='+python', type=('build', 'run'))
```

Comment:
According to the `setup.py` it's Python 2.6+

Reply:
I do not actively test for Python < 3 anymore. It _was_ supported. I'll probably update the setup.py